backend/config/curation-api.yml

```diff
@@ -831,6 +824,11 @@ components:
           "$ref": "#/components/schemas/dataset_organism"
         processing_status:
           "$ref": "#/components/schemas/processing_status_dataset"
+        processing_status_detail:
+          type: string
+          description: |
+            This field will be populated if an error that prevents the dataset from being fully processed.
```

Comment:
@Bento007 


Per the requirement

> Add dataset.processing_status_detail for VALIDATION_FAILURE to report the validation failures.

How about:

When processing_status is set to VALIDATION_FAILURE, further details about the validation failures for the dataset are returned in processing_status_detail. 

or

When processing_status is set to VALIDATION_FAILURE, further details about the validation failures for the dataset can be found in processing_status_detail.